modules/data-loading/pages/load-from-iceberg.adoc

```diff
@@ -1 +1,306 @@
+
+
+[cols="1,2,1,1",options="header"]
+|===
+| Configuration Property
+| Description (Token)
+| Required (Token)
+| Required (Credential Refresh)
+
+| iceberg.catalog.oauth2.token
+| Your bearer token
+| Yes
+| No
+
+| iceberg.catalog.oauth2.token.expires
+| Time before refresh (defaults to 300,000 ms)
+| No
+| No
+
+| iceberg.catalog.oauth2.token.refresh.enabled
+| Enable refresh (false by default)
+| Yes (false)
+| Yes (true)
+
+| iceberg.catalog.oauth2.token.endpoint.url
+| Token endpoint URL
+| No
+| Yes
+
+| iceberg.catalog.oauth2.client.id
+| Client ID for credentials
+| No
+| Yes
+
+| iceberg.catalog.oauth2.client.secret
+| Client secret for credentials
+| No
+| Yes
+
+| iceberg.catalog.oauth2.token.scopes
+| Optional scopes (defaults to null)
+| No
+| No
+|===
+
+=== MinIO Example
+Connect to MinIO with a REST catalog:
+
+[source,gsql]
+CREATE DATA_SOURCE s1 = """
+{
+    type: iceberg,
+    iceberg.catalog.type: rest,
+    iceberg.catalog.uri: http://rest:8181,
+    aws.s3.endpoint: http://minio:9000,
+    aws.s3.access_key: accesskey,
+    aws.s3.secret_key: password,
+    aws.client.region: us-east-1
+}""" FOR GRAPH socialNet
+
+=== S3 Example
+Connect to an S3 bucket:
+
+[source,gsql]
+CREATE DATA_SOURCE s1 = """
+{
+    type: iceberg,
+    iceberg.catalog.type: rest,
+    iceberg.catalog.uri: http://localhost:8181,
+    aws.s3.warehouse: s3://bucket-containing-iceberg-tables/,
+    aws.s3.access_key: accesskey,
+    aws.s3.secret_key: password,
+    aws.client.region: us-west-1
+}""" FOR GRAPH socialNet
+
+== Set Up Your Loading Job
+
+Create a loading job to turn Iceberg data into your graph’s vertices and edges. It involves defining data sources and mapping the data.
+
+=== Quick Loading Job Example
+Load social network data with filters to save time:
+
+[source,gsql]
+CREATE LOADING JOB loadSocialNet FOR GRAPH socialNet {
+  DEFINE FILENAME f1 = "$s1:SELECT personId, id, gender FROM iceberg_connector.person WHERE gender = 'male'";
+  DEFINE FILENAME f2 = "$s1:SELECT fromF, toF FROM iceberg_connector.friend";
+  DEFINE FILENAME f3 = "$s1:SELECT postId, subject, postTime FROM iceberg_connector.post WHERE postId > 50";
+  DEFINE FILENAME f4 = "$s1:SELECT fromP, toP FROM iceberg_connector.posted";
+  DEFINE FILENAME f5 = "$s1:SELECT fromL, toL, actionTime FROM iceberg_connector.liked";
+  LOAD f1 TO VERTEX person VALUES ($0, $1, $2);
+  LOAD f2 TO EDGE friend VALUES ($0, $1);
+  LOAD f3 TO VERTEX post VALUES ($0, $1, $2);
+  LOAD f4 TO EDGE posted VALUES ($0, $1);
+  LOAD f5 TO EDGE liked VALUES ($0, $1, $2);
+}
+
+=== Boost Performance
+- *2 Tasks from Data Source*: Use 2 tasks for smaller datasets.
+[source,gsql]
+CREATE DATA_SOURCE s1 = """
+{
+  type: iceberg,
+  iceberg.catalog.type: rest,
+  iceberg.catalog.uri: http://rest:8181,
+  aws.s3.endpoint: http://minio:9002,
+  aws.s3.access_key: admin,
+  aws.s3.secret_key: password,
+  aws.client.region: us-east-1,
+  tasks.max: 2
```

Comment:
In our current documentation, `tasks.max` is a filename parameter and can only be configured when defining a `FILENAME`.  
Is the ability to define `tasks.max` in a `DATA SOURCE` a new or old feature? Does it apply to all connectors or just Iceberg?
Does it apply to all filename parameters or just one (or a few)? Which ones?

This is the problem with documenting by "example". You actually create more questions than you answer. The better approach is to briefly explain the feature (or option) and then show an example. Or, show an example and then explain it. At some point, provide all the important details for what is/isn't supported.